src/components/ImageRows.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC } from "react"
+import { LabelMd } from "./Typography"
+import { Image, ImageProps } from "./Image"
+import { Divider, Flex, Show, Stack } from "@chakra-ui/react"
+
+interface ImageRowData {
+  title: string
+  images: {
+    name: string
+    image: ImageProps
+  }[]
+}
+
+interface ImageRowsProps {
+  data: ImageRowData[]
+  renderImageCallback?: (imageRowData: ImageRowData) => {}
+}
+
+export const ImageRows: FC<ImageRowsProps> = ({ data }) => {
+  // TODO: Add horizontal line after each row (except last one)
+  return (
+    <>
+      {data.map((tbtcPartnerData, index, array) => {
```

Comment:
IMO `tbtcPartnerData` doesn't fit here because it's a shared component, so it could probably be anything, not just tBTC partners. Can we rename to `item` ?

Reply:
fe0e0b66a550cb2f18a4854bbb72f648bacd49e4